packages/docs/package.json

```diff
@@ -16,10 +16,10 @@
     "vue": "^3.3.8"
   },
   "scripts": {
-    "typedoc": "pnpm -C $(git rev-parse --show-toplevel) build:editor && typedoc --options $(git rev-parse --show-toplevel)/typedoc.json",
+    "typedoc": "typedoc --options $(git rev-parse --show-toplevel)/typedoc.json",
     "dev": "pnpm run typedoc && vitepress dev --port 5200",
     "build": "pnpm run typedoc && vitepress build",
-    "build:vercel": "pnpm run typedoc && vitepress build",
+    "build:vercel": "pnpm -C $(git rev-parse --show-toplevel) build:editor && git remote add origin https://github.com/toeverything/blocksuite && pnpm run build",
```

Comment:
Any alternative to `git rev-parse --show-toplevel`? This looks a bit verbose.

Reply:
I found a non-documented option in pnpm: `pnpm run -w build:editor` which can replace `pnpm -C $(git rev-parse --show-toplevel) build:editor`.
However, for typedoc's one `typedoc --options $(git rev-parse --show-toplevel)/typedoc.json`, we might need to introduce additional package to solve that.